third_party/mlir/BUILD

```diff
@@ -2456,12 +2456,13 @@ filegroup(
 
 exports_files(
     [
-        "include/mlir/Dialect/StandardOps/Ops.td",
```

Comment:
I'm confused why these changes are necessary for this CL?  Did this make it in by accident or did I miss something?